Enable source code context for Flutter (#7157)
getsentry/sentry-docs: src/wizard/flutter/index.md

}
```

> Check out [the documentation](/platforms/flutter/performance/instrumentation/) to learn more about the API and automatic instrumentations.

Reviewer: ```suggestion
To learn more about the API and automatic instrumentations, check out the [performance documentation](/platforms/flutter/performance/instrumentation/) .
```

## Debug Symbols

> Debug symbols provide information that Sentry displays on the **Issue Details** page to help you triage an issue. We offer a range of methods to provide Sentry with debug symbols.

Reviewer 2: ```suggestion
We offer a range of methods to provide Sentry with debug symbols so that you can see symbolicated stack traces and triage issues faster.
```

> With default settings, complete stack traces are available in your Dart error, out of the box, unless you use [`split-debug-info`](https://flutter.dev/docs/perf/app-size#reducing-app-size) and [`obfuscate`](https://flutter.dev/docs/deployment/obfuscate). In those cases, you need to [upload the debug information files](/platforms/flutter/upload-debug/) generated by the `flutter` build, so Sentry can show you proper stack traces.

Reviewer: ```suggestion
With default settings, complete stack traces are shown for your Dart error out of the box. If you use [`split-debug-info`](https://flutter.dev/docs/perf/app-size#reducing-app-size) and [`obfuscate`](https://flutter.dev/docs/deployment/obfuscate), you need to [upload the debug information files](/platforms/flutter/upload-debug/) generated by the `flutter` build, so Sentry can show proper stack traces.
```

> For iOS, macOS and Android NDK native crashes, it is required to [upload the debug symbols](/platforms/flutter/upload-debug/) generated by the `flutter` build.

Reviewer: ```suggestion
For iOS, macOS and Android NDK native crashes, it is also required to [upload the debug information files](/platforms/flutter/upload-debug/) generated by the `flutter` build.
```

## Source Context

If Sentry has access to application source code, it can show snippets of code around the location of stack frames. We refer to these snippets as _source context_.

> Source context can be very useful for quickly identify problematic code.

Reviewer: ```suggestion
Source context can be very useful to quickly locate problematic code.
```

> 
> If Sentry has access to application source code, it can show snippets of code around the location of stack frames. We refer to these snippets as _source context_.
> 
> Source context can be very useful for quickly identify problematic code.

Reviewer 2: ```suggestion

If Sentry has access to your application's source code, it can show snippets of code (_source context_) around the location of stack frames, which helps to quickly pinpoint problematic code. 

```

> To enable source context, you need to upload debug symbols to Sentry. You can do that by following the [Uploading Source Code Context for Flutter Android, iOS, and macOS](/platforms/flutter/upload-debug/#uploading-source-code-context-for-flutter-android-ios-and-macos) guide.

Reviewer 2: ```suggestion
To enable source context, you'll need to upload debug symbols to Sentry by following the [Uploading Source Code Context for Flutter Android, iOS, and macOS](/platforms/flutter/upload-debug/#uploading-source-code-context-for-flutter-android-ios-and-macos) guide.
```
